mesonbuild/coredata.py

```diff
@@ -201,19 +201,19 @@ def get_builtin_option_default(optname):
         raise RuntimeError('Tried to get the default value for an unknown builtin option \'%s\'.' % optname)
 
 builtin_options = {
-        'buildtype'         : [ UserComboOption, 'Build type to use.', [ 'plain', 'debug', 'debugoptimized', 'release', 'minsize' ], 'debug' ],
-        'strip'             : [ UserBooleanOption, 'Strip targets on install.', False ],
-        'unity'             : [ UserBooleanOption, 'Unity build.', False ],
-        'prefix'            : [ UserStringOption, 'Installation prefix.', default_prefix() ],
-        'libdir'            : [ UserStringOption, 'Library directory.', default_libdir() ],
-        'libexecdir'        : [ UserStringOption, 'Library executable directory.', default_libexecdir() ],
-        'bindir'            : [ UserStringOption, 'Executable directory.', 'bin' ],
-        'sbindir'           : [ UserStringOption, 'System executable directory.', 'sbin' ],
-        'includedir'        : [ UserStringOption, 'Header file directory.', 'include' ],
-        'datadir'           : [ UserStringOption, 'Data file directory.', 'share' ],
-        'mandir'            : [ UserStringOption, 'Manual page directory.', 'share/man' ],
-        'infodir'           : [ UserStringOption, 'Info page directory.', 'share/info' ],
-        'localedir'         : [ UserStringOption, 'Locale data directory.', 'share/locale' ],
+        'buildtype':   [ UserComboOption, 'Build type to use.', [ 'plain', 'debug', 'debugoptimized', 'release', 'minsize' ], 'debug' ],
```

Comment:
__[flake8]__
* [E126] continuation line over-indented for hanging indent
* [E241] multiple spaces after ':'
* [E201] whitespace after '['
* [E202] whitespace before ']'

<!-- Comment by [SideCI](https://sideci.com) -->